packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMainInput.tsx

```diff
@@ -163,6 +183,9 @@ export function TransferPanelMainInput(props: TransferPanelMainInputProps) {
           )}
         >
           <TransferPanelInputField {...rest} />
+          {typeof inputCollapseOnClick !== 'undefined' && (
+            <CollapseInputButton onClick={inputCollapseOnClick} />
```

Comment:
I am not sure if we should have the Close button within the input field beside max button. I think it should be outside of the input, shrinking the field a by a little bit - we must consult design team to finalize this.